modules/client/src/allowlistVoting/internal/methods.ts

```diff
@@ -0,0 +1,470 @@
+import { BigNumber } from "@ethersproject/bignumber";
+import {
+  ContractReceipt,
+  Event as EthersEvent,
+} from "@ethersproject/contracts";
+import { AllowlistVoting } from "../client";
+import {
+  ICreateProposalParams,
+  ISetConfigurationParams,
+  IVoteParams,
+  Steps,
+  Proposal,
+  ProposalCreationValue,
+  VoteStepsValue,
+} from "../interfaces";
+import { arrayify } from "@ethersproject/bytes";
+
+export class AllowlistVotingMethods {
+  private allowlistVoting: AllowlistVoting;
+
+  constructor(allowlistVoting: AllowlistVoting) {
+    this.allowlistVoting = allowlistVoting;
+  }
+
+  /**
+   * Returns the hash for the MODIFY_ALLOWLIST_PERMISSION permission
+   *
+   * @return {*}  {Promise<string>}
+   * @memberof AllowlistVotingMethods
+   */
+  public MODIFY_ALLOWLIST_PERMISSION_ID(): Promise<string> {
```

Comment:
Atm, we have a constant with the permissions so we dont have to await to get a permission, we can move them to this way, but if we do it I'd also do it in the client

Reply:
moved them out